### PR TITLE
validate ipvs scheduler

### LINKED ISSUE #53975: Should validate ipvs sheduler options
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:

IPVS sheduler options has no validation. For example, if we pass an unsupported `--ipvs-scheduler` option(e.g. `foo`) for kube-proxy, it will cause unexpected error.

**What you expected to happen**:

Should validate ipvs sheduler options.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration**:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

/sig network

/area kube-proxy

### PR DESCRIPTION
**Pull request by m1093782566**
**What this PR does / why we need it**:

validate ipvs scheduler options

**Which issue this PR fixes**: 

closes #53975

**Special notes for your reviewer**:

It depends on work of #53780.

**Release note**:

```release-note
NONE
```

/sig network

/area kube-proxy

### CODE REVIEW

File: cmd/kube-proxy/app/validation.go [hunk 1]
```diff
@@ -156,3 +157,33 @@ func validateHostPort(input string, fldPath *field.Path) field.ErrorList {
...
+		componentconfig.WeightedRoundRobin,
+		componentconfig.LeastConnection,
+		componentconfig.WeightedLeastConnection,
+		componentconfig.LocalityBasedLeastConnection,
+		componentconfig.LocalityBasedLeastConnectionWithReplication,
+		componentconfig.SourceHashing,
+		componentconfig.DestinationHashing,
+		componentconfig.ShortestExpectedDelay,
+		componentconfig.NeverQueue,
+		"",
```
Comment (dixudx) [inline cmd/kube-proxy/app/validation.go]: Why empty string here?

Comment (m1093782566) [reply cmd/kube-proxy/app/validation.go]: empty string is a valid input.

**Comment by cmluciano** [comment]
/retest

no available resources at the time of test run

**Pull request merged**